proto/movementlabs/protocol_units/da/light_node/v1beta2.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+package movementlabs.protocol_units.da.light_node.v1beta2;
+
+
+// Request and response messages
+message Blob {
+    bytes blob_id = 1;
+    bytes data = 2;
+    uint64 height = 3;
+    bytes signature = 4;
+    uint64 timestamp = 5;
+    bytes signer = 6;
+}
+
+message BlobResponse {
+    oneof blob_type {
+      Blob passed_through_blob = 1;
+      Blob sequenced_blob_intent = 2;
+      Blob sequenced_blob_block = 3;
+      bool heartbeat_blob = 4;
```

Comment:
Adding a new field should be a backward-compatible change in protobuf, so this change alone does not justify the `.v1beta2` package.

Reply:
It has been discussed here: https://github.com/movementlabsxyz/movement/pull/996#pullrequestreview-2559597210
And we go to a new version of the protocol.

Reply:
If you send a heartbeat with an old client it's not compatible. I think it's more theto define a revision in the protocol.